internal/view/exec.go

```diff
@@ -378,6 +378,9 @@ func k9sShellPod(node string, cfg *config.ShellPod) *v1.Pod {
 	if len(cfg.Args) > 0 {
 		c.Args = cfg.Args
 	}
+	if len(cfg.ImagePullPolicy) > 0 {
```

Comment:
No need to check the length. We should be able to assign the pullPolicy on the container declaration above. I think this is ok if nil.